src/schema/ecommerce/types/initial_offer_input_type.ts

```diff
@@ -1,19 +1,16 @@
-import {
-  GraphQLInputObjectType,
-  GraphQLNonNull,
-  GraphQLInt,
-  GraphQLID,
-} from "graphql"
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLID } from "graphql"
+import { MoneyInput } from "schema/fields/money"
+
 export const InitialOfferInputType = new GraphQLInputObjectType({
   name: "InitialOfferOrderInput",
   fields: {
     orderId: {
       type: new GraphQLNonNull(GraphQLID),
       description: "ID of order",
     },
-    amountCents: {
-      type: new GraphQLNonNull(GraphQLInt),
-      description: "Offer amount in cents",
+    offerAmount: {
```

Comment:
I still think this is called an “offer price”. An example can be found [here](https://support.saatchiart.com/hc/en-us/articles/214385258-How-do-I-submit-an-offer-): “Enter your offer price.”

That gets rid of the awkward double use of the word ‘amount’ and I also think that, because this mutation is about making an offer, you can just call the field `price`.